addons/l10n_latam_check/wizards/l10n_latam_payment_mass_transfer.py

```diff
@@ -71,21 +71,56 @@ def _create_payments(self):
         currency_id = self.check_ids[0].currency_id
 
         pay_method_line = self.journal_id._get_available_payment_method_lines('outbound').filtered(
-            lambda x: x.code == 'out_third_party_checks')
-        payment_vals = {
-                        'date': self.payment_date,
-                        'amount': sum(checks.mapped('amount')),
-                        'payment_type': 'outbound',
-                        'memo': self.communication,
-                        'journal_id': self.journal_id.id,
-                        'currency_id': currency_id.id,
-                        'payment_method_line_id': pay_method_line.id,
-                        'l10n_latam_move_check_ids': [Command.link(x.id) for x in checks]
-                    }
+            lambda x: x.code == 'out_third_party_checks'
+        )
+        partner_id = False if len(checks.mapped("partner_id")) != 1 else checks.partner_id.id
```

Comment:
@malb-odoo Actually the possible values for partner are false or company_id.partner_id because the check no longer belongs to the customer but to the company during the transfer. If false works well I don't see it wrong. Also with the check partner the _is_latam_check_transfer method would not work.